perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java

```diff
@@ -407,6 +408,11 @@ public void setSearcherBl(SearcherBl searcherBl) {
 		this.searcherBl = searcherBl;
 	}
 
+	@Override
+	public boolean isThisPerunReadOnly() {
```

Comment:
Is this method really required herte ? It's already in BeansUtils as static method and can be called from anywhere. Also, I would drop "This" in the name of method. But it's not that important.